templates/azure/fips.html

```diff
@@ -10,219 +10,225 @@
-            <a href="https://azuremarketplace.microsoft.com/en-ca/marketplace/apps/canonical.0001-com-ubuntu-pro-bionic-fips"
-               class="p-button--positive"><span>Launch Ubuntu Pro FIPS 18.04 LTS on Azure</span></a>
-            <a href="https://azuremarketplace.microsoft.com/en-ca/marketplace/apps/canonical.0001-com-ubuntu-pro-xenial-fips"
-               class="p-button--positive"><span>Launch Ubuntu Pro FIPS 16.04 LTS on Azure</span></a>
-          </p>
+  <section class="p-section--hero">
+    <div class="row">
+      <div class="col-6">
+        <div class="p-section--shallow">
+          <h1>Ubuntu Pro FIPS<br />for Azure compliance</h1>
         </div>
-        <div class="col-5 u-align--center u-vertically-center u-hide--medium u-hide--small">
-          {{ image(url="https://assets.ubuntu.com/v1/da9a1344-Microsoft-Azure-logo_stacked_transparent.png",
-                    alt="",
-                    width="350",
-                    height="100",
-                    hi_def=True,
-                    loading="auto",) | safe
-          }}
+        <p class="p-heading--5">For Azure compliance in production environments.</p>
+        <p>
+          Ubuntu Pro FIPS is the first and only FIPS 140-2 certified image for Azure. Built upon the enhanced stability
+          and security features of <a href="https://ubuntu.com/azure/pro">Ubuntu Pro</a>, Ubuntu FIPS is a critical
+          foundation for federal programs and government contractors.
+        </p>
+        <hr class="is-muted" />
+        <a href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps/canonical.0001-com-ubuntu-pro-focal-fips"
+          class="p-button--positive"><span>Launch Ubuntu Pro FIPS 20.04 LTS</span></a>
+        <a href="https://azuremarketplace.microsoft.com/en-ca/marketplace/apps/canonical.0001-com-ubuntu-pro-bionic-fips" style="padding-right: 1rem;"><span>18.04
+            LTS&nbsp;&rsaquo;</span></a>
+        <a href="https://azuremarketplace.microsoft.com/en-ca/marketplace/apps/canonical.0001-com-ubuntu-pro-xenial-fips"><span>16.04
+            LTS&nbsp;&rsaquo;</span></a>
+      </div>
+      <div class="col-6">
+        <div class="p-image-container--3-2 u-hide--small u-hide--medium">
+          <img class="p-image-container__image" src="https://assets.ubuntu.com/v1/9ec8c60f-azure-heroi.png" alt="" />
         </div>
       </div>
     </div>
   </section>
-
-  <section class="p-strip--light">
+  
+  <section class="p-section">
+    <div class="u-fixed-width p-section--shallow">
+      <hr />
+      <h2>What's in Ubuntu Pro FIPS?</h2>
+    </div>
+    <div class="row--25-75-on-large">
+      <div class="col">
+        <hr class="is-muted u-hide--small" />
+        <div class="row">
+          <div class="col-3 col-medium-2">
+            <ul class="p-list">
+              <li class="p-list__item is-ticked">
+            <h3 class="p-heading--5 u-no-padding--top">FIPS compliant by default</h3>
```

Comment:
[Vanilla](https://vanillaframework.io/docs/patterns/grid#nested-columns) suggests not to use pattern in nested cols